### DISCUSSION TITLE
kit command

### DISCUSSION THREAD

**Discussion by 0WhiteSound** [GriefPrevention/GriefPrevention#1039] [Q&A] [answered]
I have installed Essentials Plugin and when i write /kit Claim then message "No kit found" is showing

**Comment by Jikoo** [comment, marked as answer]
GriefPrevention does not create kits for you (or have any other Essentials integration), you will have to set up any kits on your own.

**Comment by 0WhiteSound** [comment]
OK Wysłane z aplikacji Poczta dla Windows 10 Od: AdamWysłano: piątek, 2 października 2020 19:01Do: TechFortress/GriefPreventionDW: 133434543; AuthorTemat: Re: [TechFortress/GriefPrevention] kit command (#1039) Closed #1039.—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by RoboMWM** [comment]
Btw, anyone who's wondering, he said "OK"

Idk how he botched up the email.
